R/CredibilityInterval.R

```diff
@@ -33,10 +33,19 @@ CredibilityInterval <- function(inputs, fit.to.data = NULL) {
   
   excel.output <- GetExcelOutput(ci_data$projection, ci_data$fit.to.data, ci_data$inputs)
   gplot <- GetPdfOutput(ci_data$fit.extended, ci_data$projection, ci_data$inputs)
-  # invisible(list(fit.to.data = fit.to.data, fit.extended = fit.extended, projection = projection, gplot = gplot, excel.output = excel.output, inputs = inputs.copy))
-  invisible(c(
-    ci_data, gplot = gplot, excel.output = excel.output
-  ))
+  invisible(
+    list(
```

Comment:
@joshuaschwab thanks for the catch! It should output the expected set of objects now:

```
> names(z)
[1] "fit.to.data"  "fit.extended" "projection"   "gplot"        "excel.output" "inputs"    
```

Reply:
Thanks, that looks better, but I see`z$inputs` is NULL

Reply:
Ah, it should be working now after the last push! I had to slightly rearrange the internal functions to be more shiny-friendly and got a few things mixed up. It should return the same values as the the main branch on your repo now.

Reply:
Thanks!